<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>test</string>
    </property>
   </widget>
   <addaction name="menutest"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOverView">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/icons/serial-pot-16.png</normaloff>:/icons/serial-pot-16.png</iconset>
   </property>
   <property name="text">
    <string>OverView</string>
   </property>
   <property name="toolTip">
    <string>Over View</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/icons/connect-24.png</normaloff>:/icons/connect-24.png</iconset>
   </property>
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionAddNewChannel">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/icons/bullet-green-16.png</normaloff>:/icons/bullet-green-16.png</iconset>
   </property>
   <property name="text">
    <string>AddNewChannel</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
